core/src/main/java/feast/core/job/task/CreateJobTask.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2018-2020 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package feast.core.job.task;
+
+import feast.common.logging.AuditLogger;
+import feast.common.logging.LogEvent;
+import feast.common.logging.LogResource;
+import feast.common.logging.LogResource.ResourceType;
+import feast.core.job.JobManager;
+import feast.core.model.Job;
+import feast.core.model.JobStatus;
+import lombok.extern.log4j.Log4j2;
+import org.apache.logging.log4j.Level;
+
+/** Task that starts recently created {@link Job} by using {@link feast.core.job.JobManager}. */
+@Log4j2
+public class CreateJobTask extends JobTask {
+
+  public CreateJobTask(Job job, JobManager jobManager, AuditLogger auditLog) {
+    super(job, jobManager, auditLog);
```

Comment:
Why did we add this constructor?

Reply:
Originally this used the `@Builder` constructor from lombok and had fields duplicated across all 4 Job Task implementations. I move the fields up to `JobTask` abstract class however the `@builder` annotation has problems with inherited fields, hence the added constructor.